cmd/swarm/swarm-smoke/upload_and_sync.go

```diff
@@ -152,6 +173,82 @@ func trackChunks(testData []byte, submitMetrics bool) error {
 	return nil
 }
 
+// getChunksBitVectorFromHost returns a bit vector of presence for a given slice of chunks from a given host
+func getChunksBitVectorFromHost(client *rpc.Client, addrs []storage.Address) (string, error) {
```

Comment:
`getChunksBitVectorFromHost` and `getBzzAddrFromHost` really don't belong here - there should be a package which provided a type-safe `bzz client`, which has all APIs exposed in `bzz` configured in it, but since we don't have one, I am defining them here.

We might want to add this once we start using these APIs a bit more often, but for now, I think its fine for this to live here.